.github/workflows/maven.yml

```diff
@@ -27,5 +27,5 @@ jobs:
         username: ${{ github.repository_owner }}
         password: ${{ secrets.CR_PAT }}
     - name: Build with Maven
-      run: mvn -B deploy --file pom.xml -Djib.to.image=ghcr.io/sijoma/food-app/food-app-backend
+      run: mvn -B deploy --file pom.xml -Djib.to.image=ghcr.io/sijoma/food-app/food-app-backend:latest
```

Comment:
You should be able to replace :latest with.  **:0.0.$GITHUB_RUN_NUMBER**

Reply:
Just do it.